2020/hamming-codes/spanish/sentence_translations.json

```
@@ -1289,7 +1289,7 @@
  {
   "input": "As the second part of the exercise, let's have you play the role of the receiver.",
   "model": "nmt",
-  "translatedText": "Como segunda parte del ejercicio, hagamos que usted desempeñe el papel de receptor.",
+  "translatedText": "Como segunda parte del ejercicio, hagamos que tú te desempeñes como receptor.",
```

I feel that this way of saying it is quite unnatural to me, I would have written something like 
```suggestion
  "translatedText": "Como segunda parte del ejercicio, hagamos que tú seas el receptor.",
```
Any ideas on how to improve this?

---

Me gustó el cambio, más sencillo